Doc/library/os.rst

```diff
@@ -1659,6 +1659,27 @@ or `the MSDN <https://msdn.microsoft.com/en-us/library/z0kc8e3z.aspx>`_ on Windo
       :exc:`InterruptedError` exception (see :pep:`475` for the rationale).
 
 
+.. function:: readinto(fd, buffer, /)
+
+   Read from a file descriptor *fd* into a mutable
+   :ref:`buffer protocol <bufferobjects>` *buffer*.
+
+   The *buffer* should be mutable and :term:`bytes-like <bytes-like object>`. On
+   success, returns the number of bytes read. Less bytes may be read than the
+   size of the buffer. Will retry the underlying system call when interrupted by
+   a signal. For other errors, the system call will not be retried.
```

Comment:
```suggestion
   size of the buffer. The underlying system call will be retried when
   interrupted by a signal. For other errors, it will not be retried.
```